functional_tests/steps/home_page.py

```diff
@@ -0,0 +1,17 @@
+from behave import then, when  # pylint: disable=no-name-in-module
+from behave.runner import Context
+from playwright.sync_api import expect
+
+
+@when("An external user navigates to the ONS beta site homepage")  # pylint: disable=not-callable
```

Comment:
Issue: "external" here should probably set `IS_EXTERNAL_ENV=True`, so that it's using the external environment.

Although, that might work better as a `@given`

Reply:
How does this variable work exactly? Does it have to be set at app startup time or can it be toggled at any point?

Reply:
It should be settable at any point, most of the handling is in a database router which can handle it dynamically.